templates/galaxy/config/global_host_filters.py.j2

```diff
@@ -173,9 +175,11 @@ DOMAIN_SECTIONS = {
         "annotation",
         "climate_analysis"
     ],
-    'eirene': ['chemicaltoolbox','metabolomics','interactivetools']
+    'eirene': ['chemicaltoolbox','metabolomics','interactivetools'],
 }
 
+astro_sections = ["text_manipulation", "get_data", "collection_operations", "join__subtract_and_group", "expression_tools", "send_data",
```

Comment:
maybe you could include your subdomain in the `DOMAIN_SECTIONS` dict?

Reply:
I didn't do this because I wanted to omit some default sections, like "Convert Formats", which has only biology-specific converter tools, so avoid `or` in the next condition. 
I wrote it a bit differently now, does it make sense?